config/config.go

```diff
@@ -3,15 +3,16 @@ package config
 import (
 	"autonity-oracle/helpers"
 	"autonity-oracle/types"
-	"fmt"
 	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/hashicorp/go-hclog"
 	"github.com/namsral/flag"
 	"gopkg.in/yaml.v2"
 	"log"
 	"os"
 )
 
 var (
+	DefaultLogVerbosity   = 2 // 0: NoLevel, 1: Trace, 2:Debug, 3: Info, 4: Warn, 5: Error
 	DefaultGasTipCap      = uint64(1)
 	DefaultAutonityWSUrl  = "ws://127.0.0.1:8546"
 	DefaultKeyFile        = "./test_data/keystore/UTC--2023-02-27T09-10-19.592765887Z--b749d3d83376276ab4ddef2d9300fb5ce70ebafe"
```

Comment:
I would set the defaults of `DefaultPluginDir` and `DefaultPluginConfFile` according to the directory structure of the `tar.gz` attached to the release https://github.com/autonity/autonity-oracle/releases/tag/v0.1.2

Also maybe we can check set the default keyfile to something like `.autoracle/UTC...` ?

Reply:
resolved in 997d7f49c97fa012e19b509165f138c3b00391aa